instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware/otel_middleware.py

```diff
@@ -290,26 +293,6 @@ def process_request(self, request):
                 span, request
             )
 
-    # pylint: disable=unused-argument
-    def process_view(self, request, view_func, *args, **kwargs):
```

Comment:
Just to be clear, this runs after `process_request()`, right?

Reply:
Yes